community/pmc-tool/collector/main.py

```diff
@@ -38,8 +38,8 @@
 
```

Comment:
this file has two license headers. please remove one. ("Licensed to the Apache Software Foundation (ASF) under one... etc etc")